### ISSUE TITLE
[Usage]: About '--chat-template' parameters for model google/paligemma2-3b-ft-docci-448

### ISSUE THREAD

**Issue by llv22** [vllm-project/vllm#11471]
### Your current environment

I use [template_llava.jinja](https://github.com/vllm-project/vllm/blob/v0.6.5/examples/template_llava.jinja) to launch the model google/paligemma2-3b-ft-docci-448. Despite working, I wonder 1) how to decide on a template for a specific model and 2) whether my setting is correct for the model google/paligemma2-3b-ft-docci-448?
```text
vllm serve google/paligemma2-3b-ft-docci-448 --chat-template template_llava.jinja  --host 0.0.0.0  --port 8001 --enforce-eager --dtype auto
```
on a 

### How would you like to use vllm

I want to run inference of a [specific model](put link here). I don't know how to integrate it with vllm.

### Before submitting a new issue...

- [X] Make sure you already searched for relevant issues, and asked the chatbot living at the bottom right corner of the [documentation page](https://docs.vllm.ai/en/latest/), which can answer lots of frequently asked questions.

**Comment by DarkLight1337** [comment]
`google/paligemma2-3b-ft-docci-448` doesn't have a real chat template, so I suggest you use the dummy one which does not format the text, similar to https://github.com/vllm-project/vllm/blob/main/examples/template_vlm2vec.jinja

**Comment by llv22** [comment]
@DarkLight1337 Thanks for your information. I checked with https://github.com/vllm-project/vllm/blob/b689ada91e381ac8a1b197b5a52d7f1fa32fac05/vllm/model_executor/models/paligemma.py#L137 and https://github.com/vllm-project/vllm/blob/b689ada91e381ac8a1b197b5a52d7f1fa32fac05/vllm/model_executor/models/qwen2_vl.py#L839. For paligemma, this is no PromptReplacement in place. so that's why you said dummy template can work?

**Comment by DarkLight1337** [comment]
No, that's unrelated to the chat template. The chat template is defined on HuggingFace repo (via `chat_template.jinja` or `tokenizer_config.json`) for most models.

**Comment by ywang96** [comment]
@llv22
To clarify, the API server we provide will only work with instruct-format multimodal models. This is because we want to provide compatibility with OpenAI Vision API which's also based on `/chat/completions` (instead of `/completions` which has been deprecated by OpenAI themselves).

PaliGemma in particular was never instruct-finetuned or post trained, therefore using a chat template with it might result in unexpected behaviour. I suggest you use the model with `llm.generate` API or build a simple API server on top of it to support multimodal inference with `/completions`.
